.github/workflows/devnet-ci.yml

```diff
@@ -24,16 +24,17 @@ jobs:
           cache: 'yarn'
 
       - run: |
-          curl -L -o /tmp/foundry.tgz https://github.com/gakonst/foundry/releases/download/nightly-9c2469488c6872e5d17198555f7d8e1a80173151/foundry_nightly_linux_amd64.tar.gz
+          curl -L -o /tmp/foundry.tgz https://github.com/gakonst/foundry/releases/download/nightly/foundry_nightly_linux_amd64.tar.gz
           tar -xzvf /tmp/foundry.tgz
```

Comment:
There is a github action for this https://github.com/onbjerg/foundry-toolchain

Reply:
Georgios just opened a PR for that, I'll rebase this once that PR is merged.